### DISCUSSION TITLE
Configuration Export/Import

### DISCUSSION THREAD

**Discussion by baugarcia** [airbytehq/airbyte#37428] [Connector Questions] [answered]
Hello Everyone,

I've been looking into exporting my current Airbyte configuration (both sources and destinations, as I have hundreds of them) to have a backup and being able to rebuild Airbyte fast.

I've seen that Octavia-CLI has been discontinued and there is no more info on Github regarding this. Is there any way to export/import configuration? IaC is currently not an option, as I have everything already created and working in Airbyte. What would be the best way to do this?

Best Regards,

**Comment by jesperbagge** [comment, marked as answer]
Would it be possible to backup the postgres database? I'm thinking that since all configurations are stored to the database, a backup / restore of that might be an option.
